### PR TITLE
logging response body of bad status code responses (thanks andrew)

### PR DESCRIPTION
**Pull request by eishay**
@andrewconner

### CODE REVIEW

File: kifi-backend/modules/common/app/com/keepit/common/controller/LoggingFilter.scala [hunk 1]
```diff
@@ -39,34 +40,56 @@ class LoggingFilter() extends EssentialFilter {
...
+                  remoteServiceType = remoteServiceType,
+                  remoteHeaders = rh.headers.toSimpleMap.mkString(","),
+                  query = rh.rawQueryString,
+                  method = rh.method,
+                  currentRequestCount = countStart,
+                  url = rh.uri,
+                  body = body,
+                  statusCode = result.header.status
+                ))
+                println("==============4")
```
Comment (andrewconner) [inline kifi-backend/modules/common/app/com/keepit/common/controller/LoggingFilter.scala]: println's still intended?

File: kifi-backend/modules/common/app/com/keepit/common/controller/LoggingFilter.scala [hunk 2]
```diff
@@ -39,34 +40,56 @@ class LoggingFilter() extends EssentialFilter {
...
+              }
+              println("==============5")
+            }
+            timer.laps
```
Comment (andrewconner) [inline kifi-backend/modules/common/app/com/keepit/common/controller/LoggingFilter.scala:68]: This should probably be inside the enumerator, so we count how long it took to generate the response?

Comment (eishay) [inline kifi-backend/modules/common/app/com/keepit/common/controller/LoggingFilter.scala:68]: the full duration is logged in the access log and captured on `timer.done` the `laps` there is used in the http headers and we must send first them before sending the rest of the body so on error case they won't be accurate (though pretty close).

Comment (andrewconner) [inline kifi-backend/modules/common/app/com/keepit/common/controller/LoggingFilter.scala:68]: Ah cool. Makes sense.

**Pull request merged**